### PR TITLE
[WIP] Staging

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: k8s/helm/production.yaml [hunk 1]
```diff
@@ -57,10 +58,15 @@ accounts:
     paymaster_production_contract_address_v_0_1_0: "0xEA1Ba4305A07cEd2bB5e42224D71aBE0BC3C3f28"
     entrypoint_sandbox_contract_address_v_0_1_0: "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789"
     entrypoint_production_contract_address_v_0_1_0: "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789"
+    paymaster_funder_api_key: "pk_u_RbFw7NQBB7ZQIFJtziLE_S"
+    paymaster_funder_webhook_id: "65bbbc63fc2f925de8e1a561"
```
Comment (AliakseiMalyshau) [inline k8s/helm/production.yaml:63]: @LiorAgnin I can't see the values for:

```
  # -- Etherspot Wallet Factory Sandbox contract address
  etherspot_wallet_factory_sandbox_contract_address_v_0_1_0: ""
  # -- Etherspot Wallet Factory Production contract address
  etherspot_wallet_factory_production_contract_address_v_0_1_0: ""
```

in `production.yaml` file.

**Review by AliakseiMalyshau — COMMENTED** [review]
@LiorAgnin please see my comment.

**Review by AliakseiMalyshau — APPROVED** [review]
LGTM!

**Review by MusabShakeel576 — APPROVED** [review]
LGTM

File: k8s/helm/charts/api/templates/hpa.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ spec:
       object:
         describedObject:
           kind: Service
-          name: accounts
+          name: api
```
Comment (NikolayRn) [inline k8s/helm/charts/api/templates/hpa.yaml:39]: @AliakseiMalyshau is this ok?

**Review by NikolayRn — APPROVED** [review]
LGTM, except 1 little thing i commented

File: k8s/helm/charts/api/templates/hpa.yaml (hunk 2 above)
Comment (AliakseiMalyshau) [reply k8s/helm/charts/api/templates/hpa.yaml:39]: @NikolayRn yes, it was a mistake from my side, for RPS scaling should be specified the right service name. For API service name is `api`, not `accounts`.

Comment (AliakseiMalyshau) [reply k8s/helm/charts/api/templates/hpa.yaml:39]: You can check the others HPA's, I just copied an example from `accounts` and forgot to change the service name.

**Pull request merged**